CMakeLists.txt

```diff
@@ -179,13 +179,14 @@ if (NOT HWY_CMAKE_HEADER_ONLY)
     hwy/abort.cc
     hwy/aligned_allocator.cc
     hwy/nanobenchmark.cc
-    hwy/per_target.cc
     hwy/print.cc
     hwy/targets.cc
     hwy/timer.cc
   )
 endif()
 
+list(APPEND HWY_SOURCES hwy/per_target.cc)
```

Comment:
As an alternative, https://stackoverflow.com/questions/11801186/cmake-unable-to-determine-linker-language-with-c suggests either setting the language directly, or making Highway an "INTERFACE" library.

Reply:
Thanks. The issue is only that we still try to build a library. so we need at least one translation unit.
Not worth worrying about at this stage